<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MotoLink</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Versions</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="ltMotolink">
             <property name="text">
              <string>MotoLink: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="ltGui">
             <property name="text">
              <string>GUI: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lMotolink">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lGui">
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Sensor Mappings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="ltAFR">
             <property name="text">
              <string>AFR</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTableWidget" name="tableSensorAFR">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>330</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>370</height>
              </size>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>60</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>20</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <row>
              <property name="text">
               <string>8</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>11</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>12</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>13</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>14</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>15</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>16</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>17</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>20</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>22</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Voltage</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="4" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="5" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="6" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="7" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="8" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="9" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableWidget" name="tableSensorTPS">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>330</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>370</height>
              </size>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>60</number>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>20</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <row>
              <property name="text">
               <string>0</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>10</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>20</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>30</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>40</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>50</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>60</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>70</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>80</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>100</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Voltage</string>
              </property>
              <property name="textAlignment">
               <set>AlignHCenter|AlignVCenter|AlignCenter</set>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="4" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="5" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="6" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="7" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="8" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
             <item row="9" column="0">
              <property name="text">
               <string/>
              </property>
              <property name="flags">
               <set>ItemIsEditable|ItemIsEnabled</set>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="ltTPS">
             <property name="text">
              <string>TPS</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_fuel">
       <attribute name="title">
        <string>Fuel Map</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="tableFuel"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_afrmap">
       <attribute name="title">
        <string>AFR Map</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTableView" name="tableAfrMap"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_afrtarget">
       <attribute name="title">
        <string>AFR Target</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableView" name="tableAfrTgt"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_ignmap">
       <attribute name="title">
        <string>Ignition Map</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="tableIgnMap"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_igntarget">
       <attribute name="title">
        <string>Ignition Target</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableView" name="tableIgnTgt"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_knock">
       <attribute name="title">
        <string>Knock Map</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTableView" name="tableKnk"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuConnection"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
